lib/realtime/rpc.ex

```diff
@@ -31,22 +31,34 @@ defmodule Realtime.Rpc do
   @doc """
   Calls external node using :erpc.call/5 and collects telemetry
   """
-  @spec enhanced_call(atom(), atom(), atom(), any(), keyword()) :: any()
+  @spec enhanced_call(atom(), atom(), atom(), any(), keyword()) :: {:ok, any()} | {:error, any()}
   def enhanced_call(node, mod, func, args \\ [], opts \\ []) do
     timeout = Keyword.get(opts, :timeout, 15_000)
-    {latency, response} = :timer.tc(fn -> :erpc.call(node, mod, func, args, timeout) end)
 
-    Telemetry.execute(
-      [:realtime, :rpc],
-      %{latency: latency},
-      %{
-        mod: mod,
-        func: func,
-        target_node: node,
-        origin_node: node()
-      }
-    )
+    try do
```

Comment:
@filipecabaco got this from [Supavisor](https://github.com/supabase/supavisor/blob/9f56b879e7421a6245846d3b9a61dc81756d6afd/lib/supavisor/helpers.ex#L330).